### PR TITLE
chore: make project list headings like an accordion

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
Had to do a couple of commit message rewording due to conflicts but I don't really know what happened to those commits. Anyways conflicts are resolved and I think everything is good 😄

### CODE REVIEW

File: app/components/TaskList.tsx [hunk 1]
```diff
@@ -156,36 +127,13 @@ const TaskList: React.FC<Props> = ({ projectRevision }) => {
...
           text-indent: 15px;
         }
-        h3 {
-          font-size: 1rem;
-          line-height: 1;
-          border-bottom: 1px solid #d1d1d1;
-          text-indent: 15px;
-          padding: 10px 0 10px 0;
-          margin: 0;
-        }
```
Comment (matthieu-foucault) [inline app/components/TaskList.tsx:188]: I think the styles below for the links are not needed if we use the `Link` components from the BC gov theme.
See the `Dashboard` component, which uses the `BCGovLink` component

File: app/components/TaskListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  listItemName: string;
+  listItemMode?: string;
+  formListItemArray: Array<{
+    stepName: string;
+    linkUrl: { pathname: string; query: { projectRevision: string } };
+    formTitle: string;
+    formStatus: string | JSX.Element;
+  }>;
+  currentStep: string;
+  mode: string;
```
Comment (matthieu-foucault) [inline app/components/TaskListItem.tsx:19]:
```suggestion
  mode: "view" | "update" | "create";
```

File: app/components/TaskListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    index,
+    stepName,
+    linkUrl,
+    formTitle,
+    formStatus,
+  }) => {
+    return (
+      <li
+        aria-current={currentStep === stepName ? "step" : false}
+        className="bordered"
```
Comment (matthieu-foucault) [inline app/components/TaskListItem.tsx:44]: is this class name having any effect?

File: app/components/TaskListItem.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        key={index}
+      >
+        <Link href={linkUrl}>
+          <a>
```
Comment (matthieu-foucault) [inline app/components/TaskListItem.tsx:48]: This is where you should use a `BCGovLink`

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/components/TaskListItem.tsx (hunk 3 above)
Comment (Sepehr-Sobhani) [reply app/components/TaskListItem.tsx:44]: Actually, I didn't touch that one but yeah it adds borders at the bottom of each task list item.
Screen Shot 2022-05-10 at 4 01 51 PM

**Review by matthieu-foucault — DISMISSED** [review]
Beautiful. Just needs to be rebased

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**